crates/wasmtime/src/runtime/vm/component.rs

```diff
@@ -422,13 +465,25 @@ impl ComponentInstance {
             component_waitable_tables.push(StateTable::default());
         }
 
+        let num_error_context_tables = runtime_info.component().num_error_context_tables;
```

Comment:
I'm confused about how this chunk of code relates to the one below it.  Should one of them be deleted?